docs/assistant/security-assistant.asciidoc

```diff
@@ -9,7 +9,7 @@
 
 The Elastic AI Assistant utilizes generative AI to bolster your cybersecurity operations team. It allows users to interact with {elastic-sec} for tasks such as alert investigation, incident response, and query generation or conversion using natural language and much more.
 
-A connector for OpenAI or Azure OpenAI Service powers AI Assistant.
+AI Assistant relies on Generative AI connectors to communicate with third-party AI providers.
```

Comment:
Tiny nit: Maybe lowercase, to avoid confusion with the OpenAI connector's previous name? I know there's a "connector type" in Elastic called `Generative AI`, but this context seems to be more generally referring to connectors that process generative AI, not the Generative AI connector type. I could honestly go either way on this, though, so just something to consider!
```suggestion
AI Assistant relies on generative AI connectors to communicate with third-party AI providers.
```